src/app/app.component.ts

```diff
@@ -39,6 +46,16 @@ export class AppComponent implements OnInit, OnDestroy {
     this.isLoggedIn$ = this.trackLoginService.isLoggedIn$;
     this.dismissedLatestTOS$ = this.tosBannerQuery.dismissedLatestTOS$;
     this.dismissedLatestPrivacyPolicy$ = this.tosBannerQuery.dismissedLatestPrivacyPolicy$;
+    this.displayLoggedInTOSBanner$ = this.tosBannerQuery.displayLoggedInTOSBanner$;
+    this.userQuery.user$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((user) => {
```

Comment:
this ngUnsubscribe doesn't really do anything afaik. then again, app.component should never be destroyed so you kind of never need to unsubscribe